src/docs/admin/references/NodeSourcePolicyReference.adoc

```diff
@@ -255,40 +162,27 @@ and so on and so forth...
 
```

Comment:
Still missing params: **refreshCycleWindow**, **releaseDelay** and **threshold**